tests/model_execution/test_bin.py

```diff
@@ -99,6 +103,7 @@ def test_subprocess_raises___oasis_exception_is_raised(self):
                     create_binary_files(csv_dir, bin_dir, do_il=True)
 
     @given(standard_input_files(min_size=1), il_input_files(min_size=1))
```

Comment:
Where are these custom methods `standard_input_files`, `il_input_files` defined? If you're going to use them in other places perhaps you could put them in the tests data generation module `tests/data.py` (assume that's part of the `master`, but it is certainly something I'm using in the FM PR extensively for the tests).

Reply:
and done